HW14.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<body>
+    <script>
+        const array = [1, 2, 3, 4, 5, 6, 7];
+
+        function removeElement (arr, element) {
+            arr.splice(arr.findIndex((element) => element === 5), 1);
+            return arr;
```

Comment:
-  `return` is not need in this case 
- `arr.findIndex()` may return `-1` -> arr.splice(-1, 1) -> will remove last element 
- your function all the time will remove element with value `5`


Plz, fix your code
```
function removeElement(arr, item) {
  const index = arr.findIndex(element => element === item);

  if (index !== -1) {
    arr.splice(index, 1);
  }
}

// Or with indexOf()
function removeElement(arr, item) {
  const index = arr.indexOf(item);

  if (index !== -1) {
    arr.splice(index, 1);
  }
}

removeElement(array, 5); // [1, 2, 3, 4, 6, 7] - remove 5
removeElement(array, 3); // [1, 2, 4, 5, 6, 7] - remove 3
removeElement(array, 9); // [1, 2, 3, 4, 5, 6, 7] - the same
```